### PR TITLE
fix: Ocrvs 6356- Change condition of Number of previous births 

### PR DESCRIPTION
**Pull request by anamulhaquemollah**

### CODE REVIEW

File: src/form/birth/optional-fields.ts [hunk 1]
```diff
@@ -83,20 +83,13 @@ export const multipleBirth: SerializedFormField = {
...
-      parameters: [2]
-    }
-  ],
+  validator: [],
   mapping: getFieldMapping(
     'multipleBirth',
     certificateHandlebars.multipleBirth
   ),
-  inputFieldWidth: '64px'
+  inputFieldWidth: '64px',
```
Comment (rikukissa) [inline src/form/birth/optional-fields.ts]: Do you have a prettier plugin installed in your code editor? I think it should remove the trailing comma automatically

**Review by rikukissa — APPROVED** [review]

**Comment by rikukissa** [comment]
Looks good 👍 you can go ahead and merge this in

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ See [Releases](https://github.com/opencrvs/opencrvs-farajaland/releases) for rel
 - Deployment: Removes some deprecated deployment code around Elastalert config file formatting
 - Provisioning: Creates backup user on backup servers automatically
 - Provisioning: Update ansible Github action task version
+- Change condition of Number of previous births
```
Comment (Zangetsu101) [inline CHANGELOG.md]: @anamulhaquemollah could you move this under 1.5.0 section please?

Comment (anamulhaquemollah) [reply CHANGELOG.md]: Will do it.

**Pull request merged**